code/game/machinery/firealarm.dm

```diff
@@ -23,9 +23,9 @@
 	power_channel = AREA_USAGE_ENVIRON
 	resistance_flags = FIRE_PROOF
 
-	light_power = 0
+	light_power = 0.5
 	light_range = 3
-	light_color = COLOR_VIVID_RED
+	light_color = LIGHT_COLOR_FAINT_BLUE
```

Comment:
came here because of this. If you're going to give these a passive light power, the range needs to drop significantly. They do not justify the amount of light they produce, especially in comparison to things like apcs and air alarms.
Something like 1.5 is a pretty good sweetspot, then bring that up if there's an actual fire

Reply:
Okay I'll adjust